packages/core/src/components/tag/tag.md

```diff
@@ -1,36 +1,20 @@
-@# Tags
+@# Tag
 
 Tags are great for lists of strings.
 
 @reactExample TagExample
 
-@## CSS API
+@## Props
 
-An optional "remove" button can be added inside a tag as a
-`button.@ns-tag-remove`. The button is a separate element to support interaction
-handlers in your framework of choice.
+`Tag` is a stateless wrapper around its children with support for an optional
+close button. It also supports all HTML `<span>` props.
 
-A simple `.@ns-tag` without the remove button can easily function as a badge.
-
-@css tag
-
-@### Minimal tags
-
-Add the `.@ns-minimal` modifier for a lighter tag appearance. The translucent background color
-will adapt to its container's background color.
-
-@css tag-minimal
-
-@## JavaScript API
-
-The `Tag` component is available in the __@blueprintjs/core__ package.
-Make sure to review the [getting started docs for installation info](#blueprint/getting-started).
+@interface ITagProps
 
-Tag components render `.@ns-tag` elements with optional close buttons. Provide tag content as `children`.
+@## CSS
 
-You can provide your own props to these components as if they were regular JSX HTML elements. If
-you provide a `className` prop, the class names you provide will be added alongside of the default
-Blueprint class name.
-
-@interface ITagProps
+Create a tag with a `span.@ns-tag`. An optional "remove" button can be added
```

Comment:
Bold remove as it is an interactive UI element.

Reply:
the element doesn't actually say "remove", it's just an ✖️ . does it still need bold?

Reply:
It does not.